cms/templates/home_page.html

```diff
@@ -70,42 +54,6 @@ <h3 class="featured-product-header-title col-8">{{ product_cards_section_title }
       </div>
     </div>
   </div>
-{% else %}
-  <div class="course-section">
-    <div class="container">
-      {% if product_cards_section_title %}
-        <h1>{{ product_cards_section_title }}</h1>
-      {% endif %}
-      <ul class="course-listing card-listing">
-        {% for product in products %}
-          {% if not product.is_program %}
-            <li>
-          <div class="course-card highlight-card clickable">
```

Comment:
I don't think course-card or clickable is being used anywhere.

Reply:
both removed as well as the ones below as they were children of course-card